js/components/graph/Edge.js

```diff
@@ -72,5 +47,6 @@ Edge.propTypes = {
   /** Node that the edge is pointing to */
   target: PropTypes.string,
   /** function called when the edge's state has changed */
+  edgeStatus: PropTypes.string,
```

Comment:
Name this prop `status` instead of `edgeStatus`. It's not necessary to specify "edge" here since that's already the name of this component class.